Dockerfile

```diff
@@ -1,5 +1,5 @@
 # quay.io/ansible/creator-ee:v0.20.0
-FROM quay.io/ansible/creator-ee@sha256:72acf1476d32a7e56ec9045d8147eee010d8eecfc52590a3b8a7a620985696d4
+FROM quay.io/ansible/creator-ee@sha256:cffd700f8199306cec8d2d0eae202280d3fbf5e3f68c860b492de2eb511c25c0
```

Comment:
It seems that the image with this digest doesn't exist.
I see that [quay.io/ansible/creator-ee@sha256:bf95ae745601a9d199eeb12807be24413fc2d6488016069816ee74342c2c255b](https://quay.io/repository/ansible/creator-ee?tab=tags&tag=latest) is the latest available.
If it is a plan to use it, could you please update the commented line above with appropriate tag.

Reply:
@svor thank you for catching this, at the time we created this PR the image with that digest was there 🤷